route_following_plugin/test/test_stop_at_end_of_route.cpp

```diff
@@ -500,10 +501,3 @@ TEST_F(StopAndWaitTestFixture, CaseTen)
 
 }  // namespace route_following_plugin
 
-// Run all the tests
-int main(int argc, char** argv)
-{
-  testing::InitGoogleTest(&argc, argv);
-  auto res = RUN_ALL_TESTS();
-  return res;
-}
```

Comment:
please look at other ROS2 package's unit test main. We should still have unit test main otherwise unit test wont run. You can have only one across 2 test file cpps